cmd/kpod/info.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/docker/docker/pkg/system"
+	"github.com/ghodss/yaml"
+	"github.com/urfave/cli"
+)
+
+var (
+	infoDescription = "display system information"
+	infoCommand     = cli.Command{
+		Name:        "info",
+		Usage:       infoDescription,
+		Description: infoDescription,
+		Flags:       infoFlags,
+		Action:      infoCmd,
+		ArgsUsage:   "",
+	}
+	infoFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "debug, D",
+			Usage: "display additional debug information",
+		},
+		cli.BoolFlag{
+			Name:  "json",
+			Usage: "output as JSON instead of the default YAML",
+		},
+	}
+)
+
+func infoCmd(c *cli.Context) error {
```

Comment:
Eventually we want to move this logic up into libkpod or whatever it gets named.